### DISCUSSION TITLE
sandik adaletsiz

### DISCUSSION THREAD

**Discussion by gokhan85** [IBAX-io/go-ibax#1691] [Soru Cevap - Türkçe]
son iki günde toplam olarak sandıktan 15 puan cıktı.hic çıkmasa daha iyiydi.dalga mı geçiyorsunuz insanlarla.1 hafta önce konu başlattım githup puanı bile eklenmedi.

**Comment by Sampiyonbjk** [comment]
Aynen adaletsiz

**Comment by Bulent3407** [comment]
Hemde ne adeletsiz millet 100-100 açtı bana hep 100 altı

**Reply by alisabanoglu** [reply]
bazen sıfır puan çıkıyor. tamam bir puan olsun ama sıfır nedir ya:)))
